port/common/omrport.tdf

```diff
@@ -1013,3 +1013,42 @@ TraceExit=Trc_PRT_sysinfo_get_os_description_Exit Group=sysinfo Overhead=1 Level
 
 TraceEntry=Trc_PRT_sysinfo_os_has_feature_Entered Group=sysinfo Overhead=1 Level=5 NoEnv Template="sysinfo_os_has_feature: desc = %p, feature = %d"
 TraceExit=Trc_PRT_sysinfo_os_has_feature_Exit Group=sysinfo Overhead=1 Level=5 NoEnv Template="sysinfo_os_has_feature: returning with %zu."
+
+TraceException=Trc_PRT_isCgroupV1Available_statfs_failed Group=sysinfo Overhead=1 Level=1 NoEnv Template="isCgroupV1Available: statfs on %s failed with errno=%d"
+TraceException=Trc_PRT_isCgroupV1Available_tmpfs_not_mounted Group=sysinfo Overhead=1 Level=1 NoEnv Template="isCgroupV1Available: tmpfs is not mounted on %s"
+
+TraceException=Trc_PRT_addCgroupEntry_oom_for_cgroup_entry Group=sysinfo Overhead=1 Level=1 NoEnv Template="addCgroupEntry: failed to allocate memory for OMRCgroupEntry"
+TraceEvent=Trc_PRT_addCgroupEntry_added_new_entry Group=sysinfo Overhead=1 Level=2 NoEnv Template="addCgroupEntry: added new cgroup entry for subsystem %s and cgroup %s"

```

Comment:
Can the `noEnv` be removed from these?  `NoEnv` tracepoints have to do an expensive search to find the VMThread to get the tracebuffer.  For `TraceExceptions` this matters less and for asserts not at all.

Generally, if the code will need to get the VMThread for tracepoints, it should try to do that once and pass the thread along rather than having each tracepoint do the work.

Reply:
Looking further - everything here is `NoEnv`so while the guidance is good for OpenJ9 tracepoints, it doesn't appear to apply here in OMR.